### DISCUSSION TITLE
Are there any sample projects for CMS kit?

### DISCUSSION THREAD

**Discussion by Line4GitHub** [abpframework/abp#11926] [Q&A] [answered]
Are there any sample projects for CMS kit?

**Comment by ashkandehnavi** [comment, marked as answer]
you can use from [CMS module](https://github.com/abpframework/abp/tree/dev/modules/cms-kit) by this [document](https://docs.abp.io/en/abp/latest/Modules/Cms-Kit/Index)
first you must read [modularity](https://docs.abp.io/en/abp/latest/Modules/Index) document

**Reply by Line4GitHub** [reply]
Thank you very much for your reply.

**Comment by enisn** [comment]
I saw that website is built with CMS-Kit & Lepton Theme

https://byteology.co/blogs/tutorials/

That post is also posted on community.abp.io
